lib/overcommit/git_repo.rb

```diff
@@ -35,6 +36,15 @@ def extract_modified_lines(file_path, options)
       lines
     end
 
+    # Extract the set of modified lines from a given file.
+    #
+    # @param file_path [String]
+    # @param options [Hash]
+    # @return [Set] line numbers that have been modified in file
+    def extract_modified_lines_last_commit(file_path)
```

Comment:
There doesn't appear to be a compelling need to make this a separate helper. The canonical `extract_modified_lines` method will do just fine inline as it is only consumed by helpers anyway.